core/platform/email/mailgun_email_services.py

```diff
@@ -95,16 +100,17 @@ def send_email_to_recipients(
     # To send bulk emails we pass list of recipients in 'to' paarameter of
     # post data. Maximum limit of recipients per request is 1000.
     # For more detail check following link:
-    # https://documentation.mailgun.com/user_manual.html#batch-sending
+    # https://documentation.mailgun.com/docs/mailgun/user-manual/
+    # sending-messages/#batch-sending
     recipient_email_lists = [
         recipient_emails[i:i + 1000]
         for i in range(0, len(recipient_emails), 1000)]
     for email_list in recipient_email_lists:
         data = {
             'from': sender_email,
-            'subject': subject.encode('utf-8'),
```

Comment:
No need to encode manually because the request library will handle the encoding intrinsically.